includes/class_plugins.php

```diff
@@ -26,7 +26,7 @@ class class_plugins
 	var $plugins_path = '';
 	var $plugins_settings_path = '';
 
-	var $plugin_includes_array = array('constants', 'common', 'functions', 'class');
```

Comment:
I prefer keeping 'class' here. We will discuss privately the reason

Reply:
If I remember correctly, `'class'` was an addition I made that broke things.